README.md

```diff
@@ -1,12 +1,135 @@
-[node-organic](http://node-organic.com)
-============
+# [node-organic](http://node-organic.com)
+
+# Organic v1.0.0
```

Comment:
IMO @outbounder there's no clear `value definition` in the readme. Why somebody should use node-organic? What's the offering?

Research some of the trending repos https://github.com/trending?l=javascript&since=monthly to see some examples xD